astronomer/providers/amazon/aws/hooks/redshift_data.py

```diff
@@ -25,14 +27,16 @@ class RedshiftDataHook(AwsBaseHook):
     :param resource_type: boto3.resource resource_type. Eg 'dynamodb' etc
     :param config: Configuration for botocore client.
         (https://boto3.amazonaws.com/v1/documentation/api/latest/reference/core/session.html)
+    :param polling_period_seconds: polling period in seconds to check for the status
```

Comment:
what about keeping this as `poll_interval` since for most of the operators we have `poll_interval` for this?

Reply:
There are multiple places in our code base where the hooks, triggers, sensors have `polling_period_seconds` e.g. hive, gcs, bigquery, redshift_cluster, redshift_sql, databricks and snowflake.

However, the suggestion makes sense to me and I have made the change for the implementation specific to this PR.